apps/site/lib/site_web/templates/cms/page/_alerts.html.eex

```diff
@@ -0,0 +1,13 @@
+<div class="alerts-section">
+  <%
+    paths = MapSet.new([@page.path_alias | @page.redirects])
+    alerts = Enum.filter(@alerts, & alert_related?(paths, &1))
```

Comment:
Could put this all this logic in the `page_content_view.ex`, e.g. `alerts = alerts_for_page(@alerts, @page)`